<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>761</width>
    <height>607</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="btnSimpan">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>420</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Simpan</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tabelArsip">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>721</width>
     <height>151</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>ID Arsip</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Nomor Surat</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Judul Dokumen</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Tanggal</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Lokasi Penyipanan</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="btnUbah">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>420</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Ubah</string>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>571</width>
     <height>171</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="iDAnggotaLabel">
      <property name="text">
       <string>ID Arsip</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="editID"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="namaAnggotaLabel">
      <property name="text">
       <string>Nomor Surat</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="editNomor"/>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="editTanggal"/>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="editJudul"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="statusLabel">
      <property name="text">
       <string>Judul Dokumen</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="telpLabel">
      <property name="text">
       <string>Tanggal</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="emailLabel">
      <property name="text">
       <string>Lokasi Penyimpanan</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="editLokasi"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btnHapus">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>420</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Hapus</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="editCari">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>571</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnPrint">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>420</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Print</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
